.github/images/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM centos:7
+
+ENV PGHOST=postgres
+ENV PGUSER=foreman
+ENV PGPASS=foreman
+ENV RAILS_ENV=test
+ENV GIT_COMMITTER_NAME="gh_actions"
+ENV GIT_COMMITTER_EMAIL="gh_actions@rh_cloud.foreman"
+ENV WORKDIR=/projects/foreman
+ENV BUNDLE_PATH=vendor/bundle
+
+ARG SCLS="rh-ruby25 rh-nodejs12 sclo-git25 rh-postgresql12"
+
+RUN \
+  yum install -y centos-release-scl; \
+  yum install -y systemd-devel make tar libvirt-devel rh-ruby25-ruby-devel zlib-devel libxml2-devel openssl-libs libvirt-devel nodejs rh-ruby25-rubygems automake gcc gcc-c++ kernel-devel libcurl-devel sqlite-devel rh-nodejs12-npm sclo-git25 rh-postgresql12-postgresql-devel rh-postgresql12-postgresql; \
+  source scl_source enable $SCLS; \
+  git config --global user.name $GIT_COMMITTER_NAME; \
+  git config --global user.email $GIT_COMMITTER_EMAIL; \
+  mkdir /projects; \
+  cd /projects; \
+  git clone --depth 1 --branch 2.3-stable https://github.com/theforeman/foreman.git; \
```

Comment:
for master this should be `develop` I guess and then we should change it to `3.1-stable` once it will be branched.